### ISSUE TITLE
use env Render doesn't work

### ISSUE THREAD

**Issue by summer908** [sealerio/sealer#2145]
### What happen?

deploy sealer 0.9.0 version，use env mode Transmission variable  doesn't work，

### Relevant log output?

```Shell
问题描述：在Clusterfile 中用env渲染，是我们应用文件读到这个变量生效，但是目前参照官方的用法，是不生效的，不知道是哪里的问题，使用方式 or bug?
```

### What you expected to happen?

![image](https://user-images.githubusercontent.com/102030092/227860124-102abd42-c027-481f-af56-c4485b4297ee.png)
期望实现env渲染成功

但不是下面这种方式（之前我们用0.6.1版本实现方式，打算弃用这个方式）
![image](https://user-images.githubusercontent.com/102030092/227860800-79b6afb7-b6d4-4cf1-be3b-7fa5e35441b4.png)

### How to reproduce it (as minimally and precisely as possible)?

```YAML
Clusterfile:
apiVersion: sealer.cloud/v2
kind: Cluster
metadata:
  name: my-cluster
spec:
  env:
  - SkipPreflight=true
  - pc_modules=base,label
  image: localhost/test:v27
  ssh:
    passwd: xxxxx
    user: root
  hosts:
    - ips: [ 192.168.5.21 ]
      labels:
        aquila.apes: true
        master-install: true
      roles: [ master ]
    - ips: [ 192.168.5.22,192.168.5.23,192.168.5.24 ]
      roles: [ node ]
---
apiVersion: sealer.aliyun.com/v1alpha1
kind: Plugin
metadata:
  name: zpost-install-func
spec:
  env:
  - pc_modules=base,label
  type: SHELL
  action: post-install
  'on': master-install=true
  data: |
    sh ./post-install.sh
用这个部署
```

### Anything else we need to know?

我提供的是原型，可以写个脚本，echo $env去测试我说的的这个问题，
辛苦尽快验证，给个答复---

### What is the version of Sealer you using?

sealer v0.9.0

### What is your OS environment?

centos7.9

### What is the Kernel version?

3.10.0-1160.83.1.el7.x86_64

### Other environment you want to tell us?

ali clond ECS 
test cluster

**Comment by Stevent-fei** [comment]
@summer908 If you want to use it in the script, you can add `export test=xxx`

Kubefile is modified to：

```shell
Clusterfile:
apiVersion: sealer.cloud/v2
kind: Cluster
metadata:
  name: my-cluster
spec:
  env:
  - SkipPreflight=true
  - pc_modules=base,label
  image: localhost/test:v27
  ssh:
    passwd: xxxxx
    user: root
  hosts:
    - ips: [ 192.168.5.21 ]
      labels:
        aquila.apes: true
        master-install: true
      roles: [ master ]
    - ips: [ 192.168.5.22,192.168.5.23,192.168.5.24 ]
      roles: [ node ]
---
apiVersion: sealer.aliyun.com/v1alpha1
kind: Plugin
metadata:
  name: zpost-install-func
spec:
  type: SHELL
  action: post-install
  scope: master-install=true
  data: |
    export pc_modules=base,label
    sh ./post-install.sh
```

**Comment by summer908** [comment]
> @summer908如果你想在脚本中使用它，你可以添加`export test=xxx`
> 
> kubefile修改为：
> 
> ```shell
> Clusterfile:
> apiVersion: sealer.cloud/v2
> kind: Cluster
> metadata:
>   name: my-cluster
> spec:
>   env:
>   - SkipPreflight=true
>   - pc_modules=base,label
>   image: localhost/test:v27
>   ssh:
>     passwd: xxxxx
>     user: root
>   hosts:
>     - ips: [ 192.168.5.21 ]
>       labels:
>         aquila.apes: true
>         master-install: true
>       roles: [ master ]
>     - ips: [ 192.168.5.22,192.168.5.23,192.168.5.24 ]
>       roles: [ node ]
> ---
> apiVersion: sealer.aliyun.com/v1alpha1
> kind: Plugin
> metadata:
>   name: zpost-install-func
> spec:
>   type: SHELL
>   action: post-install
>   scope: master-install=true
>   data: |
>     export pc_modules=base,label
>     sh ./post-install.sh
> ```

那样不就写死了么，我的需求是在Clusterfile实现，0.9.0版本不是可以支持么？

**Comment by Stevent-fei** [comment]
@summer908 ok, this problem is considered a bug and has been fixed in v0.9.2, you can update `sealer` version and run it.
